packages/blade/src/components/Radio/_decisions/decisions.md

```diff
@@ -0,0 +1,101 @@
+# Checkbox Decisions <!-- omit in toc -->
+
+- [Design](#design)
+- [`Radio` API](#radio-api)
+- [`RadioGroup` API](#radiogroup-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+    - [Controlled, Uncontrolled](#controlled-uncontrolled)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=13133%3A160709) to all variants of the Checkbox component
+
+## `Radio` API
```

Comment:
The whole API is similar to that of `Checkbox`, right? Only the minor differences of multi-select vs single-select?

Reply:
Yep! that's correct. (and keyboard handling which is by using arrow keys, tho handled by browser)